setup.py

```diff
@@ -214,6 +228,7 @@ def pre_setup():
 
 pre_setup()
 
+conda_build = os.path.basename(sys.argv[0]) == 'conda-build'
```

Comment:
Do you think we should PR a feature that adds a `CONDA_BUILD` variable to the name space when running? This will not be true if you symlink conda build or run it from a location not in your path like `./conda-build`